torch_ttnn/passes/lowering/to_tt_pass.py

```diff
@@ -266,15 +274,22 @@ def ReplaceMoreTtManually(gm: torch.fx.GraphModule) -> torch.fx.GraphModule:
                 # out = beta * input + alpha * (batch1 @ batch2)
                 # if beta is 0, input is ignored, and nan and inf in it will not be propogated
                 new_node = g.call_function(ttnn.matmul, args=(args[1], args[2]))
+                # Intermediate node meta is not preserved, this ensures retention
+                new_node.meta["val"] = node.meta["val"]
```

Comment:
Any ideas on how to make this a common routine for conversions to avoid duplication and make sure it is consistently used across all conversions?